examples/crude_bench.rs

```diff
@@ -127,10 +127,10 @@ async fn do_parallel_address_benchmark<R>(
 ) where
     SendFuture<(), NameableSending<Counter, ()>, R>: Future,
 {
-    let (addr, ctx) = Context::new(None);
+    let (addr, mailbox) = Mailbox::new(None);
```

Comment:
I quite like the way this reads because `Address` and `Mailbox` are counterparts so it makes sense that they are instantiated together.

Reply:
This is also much more inline with various channel APIs.